tests/Spice86.Tests/CfgGraphDumper.cs

```diff
@@ -22,7 +20,8 @@ public List<string> ToAssemblyListing(Machine machine) {
     }
 
     private List<ICfgNode> DumpInOrder(Machine machine) {
-        Dictionary<SegmentedAddress, ISet<CfgInstruction>> executionContextEntryPoints = machine.CfgCpu.ExecutionContextManager.ExecutionContextEntryPoints;
+        Dictionary<SegmentedAddress, ISet<CfgInstruction>> executionContextEntryPoints =
+            ((Core.Emulator.CPU.CfgCpu.CfgCpu)machine.Cpu).ExecutionContextManager.ExecutionContextEntryPoints;
```

Comment:
Can this cast be avoided ? Like by passing the execution context manager directly, like before